### ISSUE TITLE
Error in codegen for SPIDER

### ISSUE THREAD

**Issue by hmiomand** [preesm/preesm#36]
Files generated for SPIDER have error (from stereo app in preesm-apps folder)
```	PiSDFVertex* bo_Cost_Parallel_Work = Spider::addHierVertex(
		/*Graph*/   graph,
		/*Name*/    "Cost_Parallel_Work",
		/*Graph*/   Cost_Parallel_Work(, Hor, Vert),
		/*InData*/  5,
		/*OutData*/ 1,
		/*InParam*/ 7);
```
`Cost_Parallel_Work` is expecting a non existent first parameter (should only have two).
Should be `Cost_Parallel_Work(Hor, Vert),`.

It only happens with hierarchical graphs.

**Comment by jheulot** [comment]
Should be fixed in commit c2c271216c28c2e44283cdf8cc82265823714527
Please confirm if you have the time.

**Comment by hmiomand** [comment]
Problem fixed
